### ISSUE TITLE
sample application for debezium FSM

### ISSUE THREAD

**Issue by zenyui** [namely/chief-of-state#152]
**Is your feature request related to a problem? Please describe.**
Create a sample project using COS as a FSM to unpack debezium events

**Describe the solution you'd like**
a docker-compose sample that runs a postgres DB, debezium, kafka, and COS

**Comment by Tochemey** [comment]
@zenyui I don't think this issue is related to the open source version. Need more explanation.

**Comment by zenyui** [comment]
> @zenyui I don't think this issue is related to the open source version. Need more explanation.

It came up in an open-sourcing meeting as a great use case for COS (converting debezium topics into a nice, nested struct via COS). It solves a common, hard problem, so it's worth demonstrating as a sample app.
